docs/uml/protocol/ParticipantTriggeredByPrimaryData.puml

```diff
@@ -1,7 +1,7 @@
 @startuml
-SimScheduler -> PrimaryServiceWorker: <font color=red>!</font> TriggerWithIdMessage(\n\tActivityStartTrigger(tick),\n\t_,\n\t_)
+Scheduler -> PrimaryServiceWorker: <font color=red>!</font> TriggerWithIdMessage(\n\tActivation(tick),\n\t_,\n\t_)
 PrimaryServiceWorker -> ParticipantAgents: <font color=red>!</font> <i>ProvisionMessage</i>
-PrimaryServiceWorker --> SimScheduler: <font color=red>!</font> CompletionMessage(triggerId, None)
-SimScheduler -> ParticipantAgents: <font color=red>!</font> TriggerWithIdMessage(\n\tActivityStartTrigger(tick),\n\t_,\n\t_)
-ParticipantAgents --> SimScheduler: <font color=red>!</font> CompletionMessage(triggerId, None)
+PrimaryServiceWorker --> Scheduler: <font color=red>!</font> Completion(triggerId, None)
+Scheduler -> ParticipantAgents: <font color=red>!</font> TriggerWithIdMessage(\n\tActivation(tick),\n\t_,\n\t_)
+ParticipantAgents --> Scheduler: <font color=red>!</font> Completion(triggerId, None)
```

Comment:
Same here... (also, `triggerId`s do not exist anymore as well)

Reply:
This would work the same way as above. I'd specify `newTick` here in the last Completion, and leave it as `None` in the fourth line